apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -473,14 +430,12 @@ export class CaseService {
       .getObject(`generated/${theCase.id}/ruling.pdf`)
       .then((pdf) => this.uploadSignedRulingPdfToCourt(theCase, pdf))
       .catch(() => {
-        // The signed ruling should have been delivered to the court
-        // either directly to the court system or via email
         this.logger.info(
           `The ruling for case ${theCase.id} was not found in AWS S3`,
           { caseId: theCase.id },
         )
 
-        return true
```

Comment:
Consider logging an error now since we are failing the ruling delivery

Reply:
I decided to return `false` so that an email would be send out (Ekki tókst að vista þingbók og/eða úrskurð í máli {courtCaseNumber} í Auði....). If we throw an errror that email wont be sent out.  

What does that mean for the current implementation to fail the ruling delivery? Here I would like to try again if this fails but we have no support for that right?

Reply:
Right, we will get that soon I hope.